docs/tasks/README.md

```diff
@@ -3,6 +3,23 @@
 This section contains pages that show how to perform individual tasks.
 A task page shows how to do an operation with a short sequence of steps.
 
-## [Administer Cluster Resources](administer_cluster_resources.md)
+The tasks are designed to be followed by two personas: _batch administrators_
+and _batch users_.
 
```

Comment:
The following section includes the tasks available for the following user personas:

* Batch administrators
* Batch users